addOns/alertFilters/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Added
 - Support for the Automation Framework
+- Maintenance changes.
```

Comment:
This is not actually needed (the code being changed is the line above).
The add-on needs to depend on the new version of the automation add-on.